### ISSUE TITLE
Mesh quality bad when using images of replica-room0 dataset but processed through custom dataset pipeline

### ISSUE THREAD

**Issue by lbrianza** [autonomousvision/sdfstudio#207]
Hello, first of all, thanks for this awesome repository! Especially because it is very well documented and straightforward to use.

**Describe the bug**
The final mesh looks awesome if I use the downloaded dataset as it is, but it looks awful when I try to process the same images of the dataset using the pipeline for custom data.

**To Reproduce**
First of all, I tried to recreate the mesh of the replica-room0 dataset by using the instructions provided in the readme of the main page:
`ns-download-data sdfstudio` 
` ns-train neus-facto --pipeline.model.sdf-field.inside-outside True --pipeline.model.mono-depth-loss-mult 0.1 --pipeline.model.mono-normal-loss-mult 0.05 --vis viewer --experiment-name neus-facto-replica1 sdfstudio-data --data data/sdfstudio-demo-data/replica-room0 --include_mono_prior True` 
` ns-extract-mesh --load-config outputs/neus-facto-replica-room0/neus-facto/XXX/config.yml --output-path meshes/neus-facto-replica-room0.ply` 

And after texturization, the mesh looks great indeed:
![studiosdf_mesh](https://github.com/autonomousvision/sdfstudio/assets/6195417/629e2556-eee9-419f-ac39-05a06b00e6bf)

So now, I tried to reproduce the mesh of the same dataset, but processing the original images via the pipeline for custom dataset. Therefore, I removed everything and I only kept the *rgb.png images. Then I processed them as follows:

1) Run COLMAP:
`ns-process-data images --data data/sdfstudio-demo-data/replica-room0  --output-dir data/sdfstudio-demo-data/replica-room0 `

2) Run omnidata to produce depth and normal maps:
`export data_type="colmap"
export scene_type="indoor"
export pre_data_dir="data/sdfstudio-demo-data/replica-room0 "
export sdf_data_dir="sdfstudio/replica-room0"
python scripts/datasets/process_nerfstudio_to_sdfstudio.py \
--data-type $data_type --scene-type $scene_type \
--data $pre_data_dir \
--output-dir $sdf_data_dir \
--mono-prior \
--omnidata-path /home/lucabrianza/omnidata/omnidata_tools/torch/data/ \
--pretrained-models /home/lucabrianza/omnidata/omnidata_tools/torch/pretrained_models/`

3) Train:

`ns-train neus-facto --pipeline.model.sdf-field.inside-outside True --pipeline.model.mono-depth-loss-mult 0.1 --pipeline.model.mono-normal-loss-mult 0.05  --vis viewer --experiment-name neus-facto-replica1 sdfstudio-data --data data/sdfstudio-demo-data/replica-room0 --include_mono_prior True`

4) Extract mesh:
`ns-extract-mesh --load-config outputs/data-sdfstudio-replica-room0/neus-facto/2023-08-25_172412/config.yml --output-path meshes/replica-room0-custom.ply`

Everything works successfully, however the final mesh looks awful:
![sdfstudio_2](https://github.com/autonomousvision/sdfstudio/assets/6195417/c60bc0d4-3738-446b-9302-2668ce385be1)

I am trying to understand if this huge difference is due to some mistake from my side, or if it is just because the depth/normal maps created by omnidata are not that accurate? (if I understand correctly the original depth maps of this dataset are taken using a depth sensor?). The problem is basically that every custom dataset I am processing results in a bad mesh.

Thanks for the help!

**Comment by lbrianza** [comment]
Hi! I close the issue, I realized the mistake was on my side. In fact, when processing the images with the custom pipeline, I was accidentally setting --pipeline.model.sdf-field.inside-outside False rather than --pipeline.model.sdf-field.inside-outside True, which I think is totally wrong for indoor scenes. Now setting this option correctly to True results in a mesh identical to the original.
